cmd/install.go

```diff
@@ -675,6 +712,21 @@ func openPort(ctx context.Context, portList []string) (err error) {
 	return nil
 }
 
+func showOpenPortGuideline(ctx context.Context, portList []string) {
+	log.WithContext(ctx).Warn(" - Open ports:")
+
+	for k := range portList {
+		switch utils.GetOS() {
+		case constants.Linux:
+			log.WithContext(ctx).Warnf("   sudo ufw allow %s", portList[k])
```

Comment:
space is redundant, doesn't it ?

Reply:
No, need this indentation spaces to show instruction steps like:
```
- Config ports:
       sudo ufw allow 123
       sudo ufw allow 456
```